edge-hub/core/test/Microsoft.Azure.Devices.Routing.Core.Test/checkpointers/CheckpointerTest.cs

```diff
@@ -203,35 +217,56 @@ public void TestQueueLengthMetricSetAndDecrement()
             t1.Join();
             t2.Join();
 
-            Assert.Equal(0u, Checkpointer.Metrics.GetQueueLengthCounter("endpoint", 0));
+            Assert.True(0u >= Checkpointer.Metrics.GetQueueLength("endpoint", 0));
         }
 
         [Fact]
         [Unit]
         public void TestQueueLengthMetricIncrement()
```

Comment:
Not sure why, but it seems like maybe in CI these tests don't actually use the real metrics provider but maybe are using the NullMetricsProvider... Any ideas on this? I am looking into test infrastructure now to find out if there is something different for CI than what I see locally.